### PR TITLE
Update to Protocol 14.1.0

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Updates Protocol to v14.1.0 with a bunch of small tweaks to a few recently-added components (namely Split and Logo/Wordmark). This also touches a bunch of pages to tidy up some of the temporary customizations we made in bedrock while the changes were being made upstream in Protocol.

## Testing
You'll need to run `yarn` or `make build` to update local dependencies.

On demo at: https://www-demo4.allizom.org

Successful test run: https://gitlab.com/mozmeao/www-config/-/pipelines/374690221

### CODE REVIEW

**Comment by alexgibson** [comment]
Now that https://github.com/mozilla/protocol/issues/687 is fixed, I think you should be able to remove this file: https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json#L1508

**Review by alexgibson — CHANGES_REQUESTED** [review]
I took a walk through the effected pages and everything looks good. Just one scrolling issue on the `/firefox/more/` page. r+wc 🕺 

Needs a rebase also.

File: bedrock/firefox/templates/firefox/more.html [hunk 1]
```diff
@@ -18,11 +18,12 @@
   {% call split(
     image_url='img/firefox/privacy/promise/privacy-hero.png',
     include_highres_image=True,
-    block_class='privacy-promise-hero mzp-l-split-hide-media-on-sm-md',
-    theme_class='mzp-t-dark'
+    block_class='page-hero mzp-l-split-hide-media-on-sm-md mzp-t-split-nospace',
```
[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/firefox/templates/firefox/more.html:21]: I ran into this same issue on /firefox/privacy/promise/ which uses the exact same hero, so I can do the same fixes here.

**Comment by craigcook** [comment]
Rebased and updated. The /firefox page could probably use a more extensive overhaul but maybe we can at least do some  visual cleanup in the near future.

**Review by alexgibson — APPROVED** [review]

**Pull request merged**